Source/DafnyCore/Compilers/InternalCompilersPluginConfiguration.cs

```diff
@@ -14,7 +14,7 @@ internal class InternalCompilersPluginConfiguration : Plugins.PluginConfiguratio
       new PythonBackend(options),
       new CppCompilerBackend(options),
       new LibraryBackend(options),
-      new DafnyBackend(options),
+      new RustBackend(options),
```

Comment:
Why removing the Dafny Backend?

Reply:
The "Dafny backend" no longer spits out a text representation of the program, but rather a desugared AST that can be consumed by other backends implemented in Dafny (i.e. Rust). So it's only meant for use internally by `DafnyExecutableBackend`.